woocommerce/woocommerce-android
# [TTPA CA/UK] Add button that will lead to a screen about TTP. Address UI changes

Closes: #9876


### Description

The PR brings changes to the payments hub screen:
* A button that will open a screen that explains TTP and its restrictions
* Changes that requests by the design
Design - UsEjcaCj3qJ5pqnXeYz7dq-fi

<img width="297" alt="image" src="https://github.com/woocommerce/woocommerce-android/assets/4923871/8109e0c7-7204-4bdd-907f-8b2a7bbf05c1">


### Testing instructions

* Use a device that supports TTP
* More -> Payments
* Notice the button and design changes

### Images/gif

<img width="297" alt="image" src="https://github.com/woocommerce/woocommerce-android/assets/4923871/f22e4e88-9656-4d76-a6d4-656d81fc7ad6">
